Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12
+FROM node:14.19.0
```

Comment:
confirm whether 14.19.0 will work or if we have to use only 14. only 14 should work @VISHNUDAS-tunerlabs

Reply:
@ankitpws CCS SDK  "node": ">=14.19.0"

Reply:
can we use node 16 in reports services also we are using 16 only